content/en/docs/setup/other_config/server-group-launch-settings/aws-ec2/launch-templates.md

```diff
@@ -189,6 +189,9 @@ Note that a number of these parameters complement each other. So, combining them
 </table>
 
 ## Use Cases & Sample API Requests
+NOTE: Replace placeholder values in:
+- parameters with custom values e.g. `account`, `credentials`, `amiName`, `iamRole`, etc. 
```

Comment:
It already seems pretty clear that `my_aws_account` is up for replacement.

Reply:
Some of us were running through an example API request and I thought this note might help.

Is the suggestion to remove the note and just replace `localhost` with `my-spinnaker-gate` ? I am good with that :)

Reply:
Yes please.